### ISSUE TITLE
Keyboard focus not being passed to PopupModal

### ISSUE THREAD

**Issue by CoopsCodes** [tcampb/react-calendly#132]
Hi,

I am having trouble getting the PopupModal take Keyboard focus, when the modal is triggered it places the modal at the very bottom of the tree and I have to tab through the whole page before it reaches the component.

I have tried creating a ref and forwardRef but the PopupModal itself doesnt seem to recognise the `focus()` etc.

Sadly I can't attach the code as we are in a closed repo, but I have not been able to locate a solution anywhere and am not sure if this is an unknown issue or if I could get some help on how to fix this.

I have followed all the docs, and even tried placing it in the App layer with context, but it is always the same problem.

Help would be awesome, as without a fix I cant pass an a11y fail into production.

**Comment by CoopsCodes** [comment]
Ok i have fixed this, so i'm closing it, but if anyone stumbles here with he same issue

i had to wrap the component in a div and link the ID to that

```
export default function CalendlyModal({ children }) {
  // I used context as the widget is triggered in multiple areas
  const { setIsCalendlyWidgetOpen, isCalendlyWidgetOpen } = useCalendly();

  const modalFocus = createRef();

  useEffect(() => {
    return isCalendlyWidgetOpen === true ? modalFocus.current.focus() : null;
  }, [isCalendlyWidgetOpen]);

  return (
    <Fragment>
      {my custom button with onClick to trigger the modal is here}
      <section id="calendly-modal" ref={modalFocus}>
        <PopupModal
          url="https://calendly.com/your-link"
          onModalClose={() => setIsCalendlyWidgetOpen(false)}
          open={isCalendlyWidgetOpen}
          rootElement={document.getElementById("calendly-modal")}
        />
      </section>
    </Fragment>
  );
}
```
